packages/markdown-cli/test/data/acceptance/omitted-acceptance-of-delivery.xml

```diff
@@ -114,15 +114,15 @@
               <w:sz w:val="24"/>
               
             </w:rPr>
-            <w:t xml:space="preserve">"Party B"</w:t>
+            <w:t xml:space="preserve">&quot;Party B&quot;</w:t>
```

Comment:
Don't sanitise this input. Whatever you put between the tags gets displayed as is.

Reply:
![Screenshot (127)](https://user-images.githubusercontent.com/59891164/130421542-161faf3d-1ee6-4663-91cf-b2d986f34d5b.png)
@algomaster99  Its working fine when I render it.